action/protocol/staking/read_state.go

```diff
@@ -184,3 +198,28 @@ func getPageOfCandidates(candidates CandidateList, offset, limit int) CandidateL
 	}
 	return res
 }
+
+func getTotalStakedAmount(ctx context.Context, sr protocol.StateReader) (uint64, string) {
```

Comment:
maybe return *big.Int instead of string in an internal fucntion

Reply:
fixed